### PR TITLE
commit for feature developing invert-replace color of a pdf for stirl…

### PR DESCRIPTION
**Pull request by Hashim0510**
Based on the discussion, I have developed the three varieties of functionality to replace text and background colors of a pdf and invert full colors of pdf.

I have created a replace-invert-color API to process the PDF. There are totally 3 options given for user.
Please take a look at the image for understanding.

image

1. Default high contrast color ---> user will have a default list of standard colors for background and text  as discussed before.
Please take a look at the image for understanding.
hight-contrast-UI

2. Custom Color  ---> user will have option to provide custom colors for text and background
Please take a look at the image for understanding.

custom-color-UI

3. Full invert --->> user can be able to invert the full colors of pdf
Please take a look at the image for understanding.

full-invert-UI

I did all corner case manual tests for all three options. I am attaching the pdf outputs with this.

Input PDF for all options: 
[test_document_complex-2.pdf](https://github.com/user-attachments/files/17237041/test_document_complex-2.pdf)

output for high contrast option: 
[high-contrast-color-output.pdf](https://github.com/user-attachments/files/17236973/high-contrast-color-output.pdf)

output for custom color option: 
[custom-color-replace-output.pdf](https://github.com/user-attachments/files/17236974/custom-color-replace-output.pdf)

output for full inversion:
[full-invert-output.pdf](https://github.com/user-attachments/files/17236981/full-invert-output.pdf)

### CODE REVIEW

File: src/main/resources/messages_en_US.properties [hunk 1]
```diff
@@ -480,6 +480,30 @@ home.removeImagePdf.title=Remove image
 home.removeImagePdf.desc=Remove image from PDF to reduce file size
 removeImagePdf.tags=Remove Image,Page operations,Back end,server side
 
+#replace-invert-color
+replace-color.title=Replace-Invert-Color
```
Comment (Frooodle) [inline src/main/resources/messages_en_US.properties:484]: Please add to en_GB file as well before merging

File: src/main/resources/templates/home.html [hunk 2]
```diff
@@ -248,11 +248,15 @@ <h5 class="card-title" th:text="#{settings.update}"></h5>
               <div
                 th:replace="~{fragments/card :: card(id='remove-image-pdf', cardTitle=#{home.removeImagePdf.title}, cardText=#{home.removeImagePdf.desc}, cardLink='remove-image-pdf', toolIcon='remove_selection', tags=#{removeImagePdf.tags}, toolGroup='other')}">
               </div>
+              <div
```
Comment (Frooodle) [inline src/main/resources/templates/home.html:251]: Please also add to navbar.html

**Review by Frooodle — CHANGES_REQUESTED** [review]

File: src/main/resources/messages_en_US.properties (hunk 1 above)
Comment (Hashim0510) [reply src/main/resources/messages_en_US.properties:484]: added as per request

File: src/main/resources/templates/home.html (hunk 2 above)
Comment (Hashim0510) [reply src/main/resources/templates/home.html:251]: added as per request

**Review by Frooodle — APPROVED** [review]
